src/rqt_plot/rosplot.py

```diff
@@ -38,8 +38,10 @@
 import threading
 import time
 
+from rclpy.qos import QoSDurabilityPolicy
 from rclpy.qos import QoSProfile
-from rosidl_runtime_py.utilities import get_message
+from rclpy.qos import QoSReliabilityPolicy
+from rqt_py_common.message_helpers import get_message_class
```

Comment:
We need to also undo the change to the imports. We're using `get_message` from `rosidl_runtime_py` instead of `get_message_class`.